### PR TITLE
Review

### PR DESCRIPTION
**Pull request by scottfeldman**
A few items I spotted doing a review.

### CODE REVIEW

File: cyrw/ioctl.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func (d *Device) tx(packet []byte) (err error) {
 
 	const PADDING_SIZE = 2
 	totalLen := uint32(whd.SDPCM_HEADER_LEN + PADDING_SIZE + whd.BDC_HEADER_LEN + len(packet))
-	if totalLen > uint32(len(buf8)) || len(packet) > 0xffff {
```
Comment (soypat) [inline cyrw/ioctl.go:84]: yeah not exactly sure what I was thinking

File: cyrw/ioctl.go [hunk 2]
```diff
@@ -471,9 +470,6 @@ func (d *Device) rxEvent(packet []byte) (err error) {
 		}
 	}()
 	// Split packet into BDC header:payload.
-	if len(packet) < whd.BDC_HEADER_LEN+72 {
```
Comment (soypat) [inline cyrw/ioctl.go:474]: It is redundant with DecodeEvent but there should still be a check so that DecodeBDCHeader does not panic.

**Review by soypat — CHANGES_REQUESTED** [review]

File: cyrw/ioctl.go (hunk 2 above)
Comment (scottfeldman) [reply cyrw/ioctl.go:474]: Then how about this?

```
if len(packet) < whd.BDC_HEADER_LEN {
    return ...
}
bdcHdr = whd.DecodeBDCHeader(packet)
```

Comment (soypat) [reply cyrw/ioctl.go:474]: That's good!

**Review by soypat — APPROVED** [review]
LGTM

**Pull request merged**